job_definitions/generate_upload_and_notify_counterpart_signature_pages.yml

```diff
@@ -96,6 +97,34 @@
             ]
           }
       }
+  
+- job:
+    name: 'generate-upload-and-notify-counterpart-signature-pages-periodic'
+    display-name: 'Generate, upload and notify counterpart signature pages for {{', '.join(s) for s,_,enabled if enabled}}'
+    project-type: pipeline
+    description: |
+        This job will run a series of scripts to
+        <ol>
+          <li>Generate framework agreement counterpart signature pages for suppliers approved for countersignature but
+              without a countersigned agreement</li>
+          <li>Upload the generated files to S3 and set the path in the relevant FrameworkAgreement record</li>
+          <li>Send an email to all active users on the supplier account that the counterpart file has been uploaded</li>
+        </ol>
```

Comment:
We have the same description on this `-periodic` job and the one above - we should probably mention that this one calls the other?